clients/src/main/java/org/apache/kafka/clients/consumer/ThreadAccessKey.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.kafka.clients.consumer;
```

Comment:
License header is missing -- you can just copy if from any existing file

Reply:
Thanks! Solved in https://github.com/apache/kafka/pull/13914/commits/8cc93e01c9a1f53fd0aeca5a5f6f8835d709babb